folly/experimental/exception_tracer/StackTrace.h

```diff
@@ -67,13 +69,16 @@ class StackTraceStack {
    * Is the stack empty?
    */
   bool empty() const {
-    return !top_;
+    return !top();
   }
 
   /**
    * Return the top stack trace, or nullptr if the stack is empty.
    */
-  StackTrace* top();
+  StackTrace* top() {
```

Comment:
There needs to be an internal function which returns a reference to the pointer, which all of the internal code uses rather than using this function.

Or all of the internal code should look like:

```
static constexpr size_t kTopIdx  = kIsDebug ? 1 : 0;
// ...
/*...*/ state_[kTopIdx] /*...*/;
```